depends/packages/bzip2.mk

```diff
@@ -6,8 +6,9 @@ $(package)_sha256_hash=ab5a03176ee106d3f0fa90e381da478ddae405918153cca248e682cd0
 $(package)_patches=Makefile.patch
 
 define $(package)_set_vars
-  $(package)_build_opts= CC="$($(package)_cc)"
-  $(package)_build_opts+=CFLAGS="$($(package)_cflags) $($(package)_cppflags) -fPIC"
+  # The -fPIC option must be on the CC not CFLAGS, because the bzip2 makefile does not respect the parameter.
+  $(package)_build_opts= CC="$($(package)_cc) -fPIC"
+  $(package)_build_opts+=CFLAGS="$($(package)_cflags) $($(package)_cppflags)"
```

Comment:
What about modifying the Makefile patch instead? Modifying CC seems dirty to me and we already patch the Makefile to respect CC.

Reply:
I think this is actually less likely to break if bzip2 is updated than if the bzip2 package is updated. Although your point of us already using a patch is a good one.